Neos.Flow/Classes/ObjectManagement/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -739,4 +763,68 @@ protected function compileStaticMethods(string $className, ProxyClass $proxyClas
             $compiledMethod->setBody('return ' . $compiledResult . ';');
         }
     }
+
+    protected function couldHaveEntityRelations(Configuration $objectConfiguration): bool
+    {
+        $result = false;
+        /** @var class-string $className */
```

Comment:
Is `class-string` on a plain `@var` something useful?

Reply:
fixed via 4da664557d9f9e96619a965b86e6351a5ded00ca